test/negative_tests.cpp

```diff
@@ -471,7 +456,7 @@ BOOST_AUTO_TEST_CASE(bad_checksum) {
                 }
 
                 if (bulk_object.offset > 0) {
-                    fseek(file, bulk_object.offset, SEEK_SET);
+                    fseek(file, (long)bulk_object.offset, SEEK_SET);
```

Comment:
Rather than updating commented out code, remove the test since even in the comments it says it is no longer applicable.